src/app/pages/survey/survey.page.ts

```diff
@@ -276,6 +280,11 @@ export class SurveyPage implements OnInit {
     });
   }
 
+  /**
+   * Called on ngOnInIt to set the variabled of the task
+   *
+   */
```

Comment:
what does this comment belong to?

Reply:
I'll do some more clean up. I believe I left some comments that are no longer needed